middleware/queue/include/queue/forward/state.h

```diff
@@ -446,7 +444,7 @@ namespace casual
 
          std::vector< std::string> memberships;
 
-         configuration::group::Coordinator group_coordinator;
+         bool enabled;
```

Comment:
Initialize the bool

Reply:
Can the whole forward group be enabled/disabled?

Reply:
Yes, which is asymmetrics with gateway I suppose. I think forward was the first one we did.

Reply:
But shouldn't we shutdown the group? Or is to complicated?

Reply:
Yeah, I'm not sure why I did it like this. I imagine we had the same discussion back then, but I can't remember.

Reply:
I don't remember us talking about having a _disabled_ process running. I do remember us talking about keeping an _enable indicator_ for each forward state that the queue-manager.

Reply:
I think you should change the samantics to shutdown a disabled group. You game?

Reply:
Aye